src/helpers/transport/query.rs

```diff
@@ -248,6 +251,6 @@ impl IpaQueryConfig {
 
 impl From<IpaQueryConfig> for QueryType {
     fn from(value: IpaQueryConfig) -> Self {
```

Comment:
Where is this used? Dont we need it for malicious also?

Reply:
good finding, I wasn't able to locate a single use of this conversion, so we can just try to delete it and see if any check fails. I spot-checked shuttle tests and benches and received no complains about removal